docs/source/api_commands.rst

```diff
@@ -177,9 +204,13 @@ camera_status
 
 Get: get_camera_status
```

Comment:
For one like this where we don't really know the why the result means, we should probably add some kind of indicator. As just putting the name of the command in the description isn't too helpful.

Would be good so other people know what to look at for helping to reverse engineer

Reply:
Agreed. I just wasn't sure what the return values actually mean so I'm not quite sure how to describe it. I just added what I could find for now with the plan to update it, but if it's not useful, I can just delete it for now.